### PR TITLE
Add Vim to the list of editors

### PR DESCRIPTION
**Pull request by DKunin**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,6 +40,12 @@ const editors = () => [{
 	bin: 'mate',
 	paths: [],
 	keywords: []
+}, {
+	id: 'vim',
+	name: 'vim',
```
Comment (sindresorhus) [inline index.js]: The name is officially `Vim`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "env-editor",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (sindresorhus) [inline package.json]: Don't bump. I'll do that when ready.

File: test.js [hunk 3]
```diff
@@ -40,6 +40,10 @@ test('.get() - TextMate', t => {
 	t.is(m.get('TextMate').id, 'textmate');
 });
 
+test('.get() - vim', t => {
```
Comment (sindresorhus) [inline test.js]: vim => Vim

File: index.js (hunk 1 above)
Comment (DKunin) [reply index.js]: Fixed

File: package.json (hunk 2 above)
Comment (DKunin) [reply package.json]: Fixed

File: test.js (hunk 3 above)
Comment (DKunin) [reply test.js]: Fixed

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**